DockerfileSlave

```diff
@@ -5,13 +5,13 @@ FROM jenkinsci/jnlp-slave:3.35-2-jdk11
 USER root
 
 RUN apt-get update \
-    && apt-get install -y apt-transport-https ca-certificates \
-    && apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D \
```

Comment:
we don't check the key anymore?

Reply:
the key is loaded on line 9 now:
```
curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -
```

Reply:
AFAIU, we fetched a known key before and now we just blindly accept the key the server gives us. It looks like a regression. Not sure it's a problem, but still, it deserves to ask if we can do like before or not

Reply:
I just copy/paster the official documentation: https://docs.docker.com/engine/install/debian/ but that's right without checking that the key matches the expected one

Reply:
this can be easily tested outside the process, but I suggest to fix this in an other issue

Reply:
and here it is: https://github.com/linagora/james-jenkins2/pull/12